### DISCUSSION TITLE
LLVM/Clang build

### DISCUSSION THREAD

**Discussion by Alessandar** [xanmod/kernel#288] [Q&A]
What's the situation in regards to the Clang config for building XanMod? Is there a technical issue? An ETA? Or is XanMod dropping support for Clang? Thanks! :smile:

**Comment by gemone** [comment]
`Xanmod` base on `kernel`. You can try :

```shell
make CC=clang defconfig

make CC=clang
```

see [Building Linux with Clang/LLVM](https://www.kernel.org/doc/html/latest/kbuild/llvm.html#building-linux-with-clang-llvm)
